src/Step/FilterStep.php

```diff
@@ -39,7 +40,7 @@ public function process($item, callable $next)
     {
         foreach (clone $this->filters as $filter) {
             if (false === call_user_func($filter, $item)) {
-                return false;
+                throw new FilterException();
```

Comment:
The thing is, i agree if we goes with good design this should throw an exception but if we decide to focus on performance this shouldn't be modified. 

Is there any alternative to throw an exception? I really like good software design, but performance is also importance.